howto/application/stream-application.md

```diff
@@ -24,7 +24,9 @@ For example, to create an instance with a 1080p resolution, a frame rate of 60 a
 
 The dashboard has in-browser streaming capabilities through WebRTC. It uses the {ref}`sec-streaming-sdk`.
 
-You can start a streaming session for any of the successfully created applications. An application is ready to be streamed when its *Status* is *Ready*. When you select the *Start a new session* icon for an application, you can specify the desired streaming attributes such as the screen resolution, frame rate, screen orientation before launching the session.
+When creating an instance, make sure you select the *Enable Streaming* capability to be able to stream your application. You can also set the desired streaming attributes using the *Virtual display* options available for an instance.
+
+You can start a streaming session for any of the successfully created applications. Once the associated instance is created and ready, click *Join session* ( ![join session icon](/images/join-session-icon.png) ) to start the stream.
```

Comment:
I'm confused, is this about the CLI or the dashboard? If so, the sessions or anything related to it doesn't exist in the dashboard anymore 🤔

Reply:
That's a good catch, I was checking the demo server when I wrote this but I checked it now and the icon tooltip says "Stream" indeed. Maybe I was looking at the wrong machine.